### ISSUE TITLE
Suggestion: Allay Overhaul

### ISSUE THREAD

**Issue by SuperStefan97** [Alex-the-666/DomesticationInnovation#199]
Allay are one of those nice viewing mobs that actually do nothing compared to the cinematic. Using a cookie could tame them, making them follow the sit/wonder/follow orders.
Tamed allay could use weapons and/or bows to defend themselves. Also, giving them a rotten apple or a sinister carrot could make them tamed Vexes, which would do the same, but me more aggresive toward threats. 
This changes would make the allays more than just a useless decoration (it can be used of course in redstone stuff... but it's very inefficient).
Finally, tamed allays could be dyed for more customization.
Thanks for reading.

**Comment by OlaPessoa** [comment]
i was literally going to talk about that while also mentioning the idea of a whole new mechanic about taming mobs, turning iron golems onto pets that whenever you are the one who built them, they have the wander, follow, stay and patrol (new mechanic) modes to stay at (which is basically going around the area you've told him to patrol and there could be enchantments to increase his range of agro)
also adding the tameable features to allay, sniffer, pandas, polar bears, turtles, phantoms and pretty much everything, this concept of being able to domesticate them came to me a few days ago when thinking of how pets work on Minecraft Dungeons, why don't we make ourselves able to domesticate mobs and instead of them being just friendly pets who do not help at all in combat, why don't we make most of the mobs actually tameable and give them each a purpose, making them actual useful pets!

**Comment by Daralexen05** [comment]
> Allay are one of those nice viewing mobs that actually do nothing compared to the cinematic. Using a cookie could tame them, making them follow the sit/wonder/follow orders. Tamed allay could use weapons and/or bows to defend themselves. Also, giving them a rotten apple or a sinister carrot could make them tamed Vexes, which would do the same, but me more aggresive toward threats. This changes would make the allays more than just a useless decoration (it can be used of course in redstone stuff... but it's very inefficient). Finally, tamed allays could be dyed for more customization. Thanks for reading.

I do not agree with this idea. This makes them yet another combat pet, which this mod and the supported Alex's Mobs already have plenty of. Lastly, Vexes die and despawn after a certain period of time. So, the "Vex" created from an Allay in this proposal would need to be a separate mob in order to prevent despawning due to being a hostile mob and to avoid being targeted by defence systems. Crows and Blue Jays already exist as flying semi-combat pets. Allays should stay as support units in my opinion.

**Comment by OlaPessoa** [comment]
> > Allay are one of those nice viewing mobs that actually do nothing compared to the cinematic. Using a cookie could tame them, making them follow the sit/wonder/follow orders. Tamed allay could use weapons and/or bows to defend themselves. Also, giving them a rotten apple or a sinister carrot could make them tamed Vexes, which would do the same, but me more aggresive toward threats. This changes would make the allays more than just a useless decoration (it can be used of course in redstone stuff... but it's very inefficient). Finally, tamed allays could be dyed for more customization. Thanks for reading.
> 
> I do not agree with this idea. This makes them yet another combat pet, which this mod and the supported Alex's Mobs already have plenty of. Lastly, Vexes die and despawn after a certain period of time. So, the "Vex" created from an Allay in this proposal would need to be a separate mob in order to prevent despawning due to being a hostile mob and to avoid being targeted by defence systems. Crows and Blue Jays already exist as flying semi-combat pets. Allays should stay as support units in my opinion.

fr, actually i do not have the slightest idea about most of the stuff you said there, but aside of that, the only reason why i would want to tame allays, is to have them as like somewhat kind of medic or support pets to follow me, instead of just fleeing away as soon as possible

**Comment by OlaPessoa** [comment]
like fr, i just saved you from being locked up for eternity, now you shall be locked with me for eternity yk?
why would i save ya if it was for the sake of giving ya freedom? that's no fun, thinking about it, Jeez minecraft makes slavery seems so normal... LMFAO
